src/features/locators/utils/copyLocatorToClipboard.ts

```diff
@@ -0,0 +1,38 @@
+import { copyToClipboard } from "../../../common/utils/copyToClipboard";
+
+export const escapeLocator = (locator: string) => {
+  let transformedText = locator.replace(/[\\'\n]/g, (match: string) => {
+    switch (match) {
+      case "\\":
+        return "\\\\\\\\";
+      case "'":
+        return "\\'";
+      case "\n":
+        return "\\n";
+      default:
+        return match;
+    }

```

Comment:
It looks like magic strings to me now. 
Is it possible for you to add an explanation or constants with meaningful names to the code explaining them?

Reply:
sorry, not my code. but it's covered with tests, hope they could help you to understand what's going on

Reply:
I just refactored location of this function

Reply:
what tests cover this code?

Reply:
`"escape symbols in locator"` from https://github.com/jdi-testing/jdn-ai/pull/1439/files#diff-c3f66e9470dbf46aa684c5ffec7394f39ed3ff49ba5e11d6aa1427f317a57938